package.json

```diff
@@ -35,6 +35,8 @@
     "exports": false
   },
   "devDependencies": {
+    "chai": "^3.5.0",
```

Comment:
I think the original `assert` library from Node.js is enough without having to introduce Chai.